src/index.ts

```diff
@@ -140,10 +151,11 @@ const runOneTest = async (
   }
   const endTime = new Date().valueOf();
   log(
-    `Finished: ${test.name} [${result.status ? "PASSED" : "FAILED"} in ${
-      (endTime - startTime) / 1000
-    } seconds]`,
-    result.status ? LogLevel.SUCCESS : LogLevel.ERROR
+    `Finished: After script in ${calculateExecutionTime(
```

Comment:
This line is incorrect, can you ensure the log statement does not change. For example all tests report `Finished: After script`

however in this case it should be

```typescript
`Finished: ${test.name} [${result.status ? "PASSED" : "FAILED"} in ${
```